lib/puppet/provider/firewalld_zone/firewall_cmd.rb

```diff
@@ -145,7 +142,7 @@ def icmp_blocks=(i)
         execute_firewall_cmd(['--remove-icmp-block', block])
       end
     end
-    unless set_blocks.empty?
+    unless set_blocks.empty? # rubocop:disable Style/GuardClause
```

Comment:
I prefer disabling this cop here to keep this block consistent with the code above it.